### DISCUSSION TITLE
May I only use nylo_framework for my flutter project?

### DISCUSSION THREAD

**Discussion by jet10000** [nylo-core/nylo#123] [General] [answered]
May I only use nylo_framework for my flutter project?  

Not started with Nylo project scaffold?

**Comment by agordn52** [comment, marked as answer]
Hi @jet10000,

You can but I would not recommend it.
It's better to use `nylo_framework` within the boilerplate provided because that's the structure the framework is built around.

Hope that answers your question 👍
